src/it/scala/org/zalando/spearheads/innkeeper/routes/AcceptanceSpecsHelper.scala

```diff
@@ -157,7 +157,7 @@ object AcceptanceSpecsHelper extends ScalaFutures with Matchers {
     makeRequest(requestUri, token, HttpMethods.DELETE)
   }
 
-  private def doGet(requestUri: String, token: String = ""): HttpResponse = {
+  def doGet(requestUri: String, token: String = ""): HttpResponse = {
```

Comment:
You should create a `getSlashHosts` method in this file, to by in sync with the overall design...